src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -80,130 +80,130 @@
-    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32-bit compatibile)</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64-bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64bit compatibile)</sys:String>
     <sys:String x:Key="Deploy_Channel">Canale aggiornamento</sys:String>
-    <sys:String x:Key="Deploy_Channel_Available_Version">Versione di Office attualmente disponibile: {0}</sys:String>
-    <sys:String x:Key="Deploy_Module">Modulo di installazione</sys:String>
-    <sys:String x:Key="Deploy_Module.Description">Scegli Office Deployment Tool per una migliore compatibilità oppure lo Strumento di Office Plus per esperienze migliori.</sys:String>
-    <sys:String x:Key="Deploy_Module.ODT">Office Deployment Tool</sys:String>
+    <sys:String x:Key="Deploy_Channel_Available_Version">Versione Office attualmente disponibile: {0}</sys:String>
+    <sys:String x:Key="Deploy_Module">Modulo installazione</sys:String>
+    <sys:String x:Key="Deploy_Module.Description">Per una migliore compatibilità scegli Strumento distribuzione Office Deployment Tool o per esperienze migliori lo Strumento di Office Plus.</sys:String>
+    <sys:String x:Key="Deploy_Module.ODT">Strumento distribuzione Office</sys:String>
     <sys:String x:Key="Deploy_Module.OTP">Office Tool Plus</sys:String>
-    <sys:String x:Key="Deploy_Download_First">Prima scaricalo, poi distribuiscilo</sys:String>
-    <sys:String x:Key="Deploy_Download_First.Description">I file di installazione verranno salvati nella directory del programma o nella directory specificata dal percorso di origine.</sys:String>
-    <sys:String x:Key="Deploy_Create_Shortcut">Crea un collegamento sul desktop</sys:String>
-    <sys:String x:Key="Deploy_Create_Shortcut.Description">Crea collegamenti sul desktop alle applicazioni di Office dopo l'installazione di Office.</sys:String>
+    <sys:String x:Key="Deploy_Download_First">Scarica i file  e poi distribuiscili</sys:String>
+    <sys:String x:Key="Deploy_Download_First.Description">I file di installazione verranno salvati nella cartella del programma o nella cartella specificata nel percorso sorgente.</sys:String>
+    <sys:String x:Key="Deploy_Create_Shortcut">Crea collegamento sul desktop</sys:String>
+    <sys:String x:Key="Deploy_Create_Shortcut.Description">Dopo l'installazione di Office crea i collegamenti sul desktop alle applicazioni di Office.</sys:String>
     <!-- Do not edit it or translate it. iSlide only provided in mainland China. -->
-    <sys:String x:Key="Deploy_iSlide">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide.Description">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install.Name">PlaceHolder</sys:String>
+    <sys:String x:Key="Deploy_iSlide">Segnaposto</sys:String>
```

Comment:
There is no need to translate line 149 to 152.

Reply:
Fixed (restored origional english strings).